Diagnostics/HealthChecker/DataCollection/ExchangeInformation/Get-ExchangeAMSIConfigurationState.ps1

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+. $PSScriptRoot\..\..\Helpers\Invoke-CatchActions.ps1
+Function Get-ExchangeAMSIConfigurationState {
+
+    begin {
+        Write-Verbose "Calling: $($MyInvocation.MyCommand)"
+    } process {
+        try {
+            Write-Verbose "Trying to query AMSI configuration state"
+            $amsiConfiguration = Get-SettingOverride | Where-Object { ($_.ComponentName -eq "Cafe") -and ($_.SectionName -eq "HttpRequestFiltering") } -ErrorAction Stop
+
+            if (($null -ne $amsiConfiguration) -and
+                ($amsiConfiguration.Count -eq 1)) {
+                Write-Verbose "Setting override detected for AMSI configuration"
+                $amsiConfigurationQuerySuccessful = $true
+                Switch ($amsiConfiguration.Parameters.Split("=")[1]) {
+                    ("False") { $amsiState = $false }
+                    ("True") { $amsiState = $true }
+                    Default { $amsiState = "Unknown" }
+                }
+
+                if ($null -eq $amsiConfiguration.Server) {
+                    $amsiOrgWideSetting = $true
```

Comment:
remove the if statement and do the following instead:

```
$amsiOrgWideSetting = $null -eq $amsiConfiguration.Server
```

Reply:
Done. Will be addressed with the next commit.